File: python/6_kyu/stop_spinning_my_words.py
# Stop gninnipS My sdroW!
# https://www.codewars.com/kata/5264d2b162488dc400000001/train/python

def spin_words(sentence):
  ans = []

  for i in sentence.split(" "):
    if len(i) >= 5:
      ans.append(i[::-1])
    else:
      ans.append(i)
  
  return " ".join(ans)


print(spin_words("Hey fellow warriors"))
print(spin_words("This is another test"))

File: javascript/6_kyu/stop_spinning_my_words.js
// Stop gninnipS My sdroW!
// https://www.codewars.com/kata/5264d2b162488dc400000001/train/javascript

function spinWords(sentence){
  var ans = [];
  sentence = sentence.split(" ");

  for (var i = 0; i < sentence.length; i++) {
    if (sentence[i].length >= 5) {
      ans.push(sentence[i].split("").reverse().join(""));
    } else {
      ans.push(sentence[i]);
    }
  }

  return ans.join(" ");
}

console.log(spinWords("Hey fellow warriors"))
console.log(spinWords("This is another test"))